src/modules/windows/notes/NotesContentsWidget.cpp

```diff
@@ -263,6 +266,20 @@ void NotesContentsWidget::triggerAction(int identifier, const QVariantMap &param
 	}
 }
 
+void NotesContentsWidget::setZoom(int zoom)
```

Comment:
I'm not sure if it is the best way to implement it, perhaps it could be achieved using `QPlainTextEdit::zoomIn()` and `QPlainTextEdit::zoomOut()`, but this API is kind of crappy. :-/
Anyway, I think that zooming should be implemented in `TextEditWidget` (adding `getZoom()` and `setZoom()`).
Also, using just point size might be risky, since AFAIR it can return 0, we might consider some method in `Utils` to apply zoom to fonts.